codegenerator/cli/npm/envio/src/Types.ts

```diff
@@ -1,6 +1,6 @@
 export type Invalid = never;
 
-export type Address = string;
+export type Address = `0x${string}`;
```

Comment:
Will this have any impact if say a function expects an address as a string but now it has a template literal type?